codebuild/samples/setup-linux.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+
+env
+
+# build java package
+cd $CODEBUILD_SRC_DIR
+
+ulimit -c unlimited
+mvn compile
+mvn install -DskipTests=true
+
+cert=$(aws secretsmanager get-secret-value --secret-id "unit-test/certificate" --query "SecretString" | cut -f2 -d":" | cut -f2 -d\") && echo -e "$cert" > /tmp/certificate.pem
```

Comment:
Is there any point in storing the results of these commands into variables?

Reply:
not really, I just copied you